src/components/form/sections/PersonalInformationSection.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import TextField from '../../common/TextField';
+
+const PersonalInformationSection: React.FC = () => {
+  return (
+    <div className="form-section">
+      <h2>Personal Information</h2>
+      <div className="form-row">
+        <TextField 
+          name="inquiryDate" 
+          label="Date of Inquiry" 
+          type="date" 
+          required 
+        />
```

Comment:
Let's remove this for now, we should default that automatically. We'll deal with that later